Enigmatry.Entry.Blueprint.Api/Enigmatry.Entry.Blueprint.Api.csproj

```diff
@@ -4,9 +4,10 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);1591</NoWarn>
     <TargetFramework>net8.0</TargetFramework>
-    <SpaRoot>..\Enigmatry.Entry.Blueprint.App</SpaRoot>
-    <SpaProxyLaunchCommand>npm start</SpaProxyLaunchCommand>
-    <SpaProxyServerUrl>https://localhost:4200</SpaProxyServerUrl>
+    <SpaRoot>..\enigmatry-entry-blueprint-app</SpaRoot>
+    <!-- Uncomment the following 2 lines to automatically start FrontEnd when debugging the API: -->
+    <!--<SpaProxyLaunchCommand>npm start</SpaProxyLaunchCommand>
+    <SpaProxyServerUrl>https://localhost:4200</SpaProxyServerUrl>-->
```

Comment:
Will everything work if we keep this uncommented but we add new launch profile entry:

`"IIS Express with Angular": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "applicationUrl": "https://localhost:44394/",
      "launchUrl": "swagger",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development",
        "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.AspNetCore.SpaProxy"
      }`